core_impl/src/test/java/io/opencensus/stats/ViewManagerImplTest.java

```diff
@@ -264,6 +335,43 @@ public void testRecordMultipleTagValues() {
         EPSILON);
   }
 
+  @Test
+  public void testRecordIntervalMultipleTagValues() {
+    // The interval is 10 seconds, i.e. values should expire after 10 seconds.
+    View view = View.create(
+        VIEW_NAME, VIEW_DESCRIPTION, MEASURE, AGGREGATIONS, Arrays.asList(KEY),
+        Interval.create(TEN_SECONDS));
+    clock.setTime(Timestamp.fromMillis(100));
+    viewManager.registerView(view);
+
+    // record for TagValue1 at 1st second
+    clock.setTime(Timestamp.fromMillis(1 * MILLIS_PER_SECOND));
+    statsRecorder.record(
+        createContext(factory, KEY, VALUE),
+        MeasureMap.builder().set(MEASURE, 10.0).build());
+
+    // record for TagValue2 at 5th second
+    clock.setTime(Timestamp.fromMillis(5 * MILLIS_PER_SECOND));
+    statsRecorder.record(
+        createContext(factory, KEY, VALUE_2),
+        MeasureMap.builder().set(MEASURE, 30.0).build());
+    statsRecorder.record(
+        createContext(factory, KEY, VALUE_2),
+        MeasureMap.builder().set(MEASURE, 50.0).build());
+
+    // get view at 12th second, stats for TagValue1 should have expired.
```

Comment:
It would also be useful to check that there are values from both tag combinations before any expire.